src/phoenix/datasets/experiments.py

```diff
@@ -267,75 +317,104 @@ def _evaluate_experiment(
         if example:
             example_run_pairs.append((deepcopy(example), exp_run))
     evaluation_inputs = [
-        (example, run, evaluator)
+        (example, run, evaluator.name, evaluator)
         for (example, run), evaluator in product(example_run_pairs, evaluators)
     ]
 
+    project_name = "evaluators"
+    resource = Resource({ResourceAttributes.PROJECT_NAME: project_name} if project_name else {})
+    tracer_provider = trace_sdk.TracerProvider(resource=resource)
+    tracer_provider.add_span_processor(
+        SimpleSpanProcessor(OTLPSpanExporter(urljoin(f"{_get_base_url()}", "v1/traces")))
+    )
+    tracer = tracer_provider.get_tracer(__name__)
+    root_span_kind = "EVALUATOR"
+
     def sync_evaluate_run(
-        obj: Tuple[Example, ExperimentRun, ExperimentEvaluator],
+        obj: Tuple[Example, ExperimentRun, ExperimentEvaluatorName, ExperimentEvaluator],
```

Comment:
What's the purpose of `ExitStack` here? It sounds like it's mainly used when you want to dynamically choose a set of context managers to run. In this case, how's it different from using two context managers?


```suggestion
        with (
            tracer.start_as_current_span(root_span_name, context=Context()) as span,
            stack.enter_context(capture_spans(resource)),
        ):
```

Reply:
doesn't work on python 3.8